python/ray/util/client_connect.py

```diff
@@ -11,7 +11,7 @@
 from ray.util.client import ray
 
 
-@Deprecated
+@Deprecated(message="Use ray.init() instead.")
```

Comment:
This message should be more descriptive and tell them exactly what to do, e.g. link to the part of the documentation that describes using the ray:// protocol with ray.init.